ui/app/models/pki/pki-issuer-engine.js

```diff
@@ -0,0 +1,51 @@
+import Model, { attr } from '@ember-data/model';
+// import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+import { withModelValidations } from 'vault/decorators/model-validations';
+
+const validations = {
+  name: [
+    { type: 'presence', message: 'Name is required.' },
+    {
+      type: 'containsWhiteSpace',
+      message: 'Name cannot contain whitespace.',
+    },
+  ],
+};
+
+@withModelValidations(validations)
+export default class PkiIssuersEngineModel extends Model {
+  @attr('string', { readOnly: true }) backend;
+  @attr('string', {
+    label: 'Issuer name',
+    fieldValue: 'id',
+  })
+  name;
+
+  // get useOpenAPI() {
```

Comment:
Waiting to confirm with backend but it does not appear that Open API exist for the issuer endpoint.